docs/configuration.rst

```diff
@@ -17,45 +17,44 @@ The images are available for Confluent Platform 3.0.1 and greater. Images are av
+| Replicator         | cp-enterprise-replicator     | Enterprise   | - confluent-kafka-replicator            |
+|                    |                              |              | - confluent-schema-registry             |
+|                    |                              |              | - confluent-control-center              |
++--------------------+------------------------------+--------------+-----------------------------------------+
+| Kafka Connect Base | cp-kafka-connect-base        | Enterprise   | - confluent-schema-registry             |
+|                    |                              |              | - confluent-control-center              |
++--------------------+------------------------------+--------------+-----------------------------------------+
+| Kafka Connect      | cp-kafka-connect             | Enterprise   | - confluent-kafka-connect-jdbc          |
+|                    |                              |              | - confluent-kafka-connect-hdfs          |
+|                    |                              |              | - confluent-schema-registry             |
+|                    |                              |              | - confluent-control-center              |
+|                    |                              |              | - confluent-kafka-connect-elasticsearch |
+|                    |                              |              | - confluent-kafka-connect-s3            |
++--------------------+------------------------------+--------------+-----------------------------------------+
+| Schema Registry    | cp-schema-registry           | Open Source  | - confluent-schema-registry             |
++--------------------+------------------------------+--------------+-----------------------------------------+
+| REST Proxy         | cp-kafka-rest                | Open Source  | - confluent-kafka-rest                  |
++--------------------+------------------------------+--------------+-----------------------------------------+
+| KSQL Server        | cp-ksql-server               | Enterprise*  | - ksql-server                           |
+|                    |                              |              | - confluent-monitoring-interceptors     |
++--------------------+------------------------------+--------------+-----------------------------------------+
+| KSQL CLI           | cp-ksql-cli                  | Open Source  | - ksql-cli                              |
++--------------------+------------------------------+--------------+-----------------------------------------+
+
+* Note: The Kafka Connect and Kafka Connect Base images are labeled as "Enterprise" simply because they contain the Confluent Control Center package.  That package enables the deployed connectors to collect the metrics visualized in Confluent Control Center.   No explicit license is required when using these images on their own.
```

Comment:
The table includes intentional changes (adding `cp-kafka-connect-base`) but most of these just look like they come from branching off of `4.x.y` docs instead of the `5.0.0-post` docs (which were created after this PR was opened). I'll update accordingly.